package.json

```
@@ -29,6 +29,7 @@
     "flow": "flow",
     "lint": "eslint .",
     "ios": "react-native run-ios --project-path \"./example/ios\"",
+    "start:macos": "node node_modules/react-native-macos/local-cli/cli.js start --use-react-native-macos",
```

[imo] confirmed that this also works, and it looks cleaner.
Although I agree that it is more accurate to use the cli inside the react-native-macos package
```suggestion
    "start:macos": "react-native start --use-react-native-macos",
```

---

Works for me.